src/model.jl

```diff
@@ -71,6 +75,12 @@ function _make_bus_demand_flexibility_amount(
         demand_flexibility.flex_amt_dn[start_index:end_index, 2:end]
     )
 
+    # percentage flexibility from input
+    if demand_flexibility.enable_doe_flexibility
+        simulation_demand_flex_amt_up .*= permutedims(bus_demand[sets.flexible_bus_idx, :])
+        simulation_demand_flex_amt_dn .*= permutedims(bus_demand[sets.flexible_bus_idx, :])
+    end
```

Comment:
This is fine for this PR, but we might need to rethink this for the forthcoming PR we've previously discussed that will allow for the mixed input of profiles (with MW values) and EIA data (with percentage values). In such a setup, we'll probably need separate variables for holding profile-specific data and EIA-specific data, whereas right now we're able to use `demand_flexibility.flex_amt_up` and `demand_flexibility.flex_amt_dn` for both situations.

Reply:
correct, I am storing the DOE flexibility data under new fields in the next version. Currently the user has to choose between DOE or input csvs so when using the DOE flexibility it also uses the old container fields.

Reply:
Okay, sounds good!